evm-test-helpers/src/subproviders/ovm-ganache.ts

```diff
@@ -0,0 +1,50 @@
+import { GanacheProvider, JSONRPCRequestPayload } from 'ethereum-types'
+// @ts-ignore
+import Ganache from 'ganache-core'
+import { Subprovider, Callback, ErrorCallback } from '@0x/subproviders'
+import { makeDebug } from '../debug'
+const ovm = require('@eth-optimism/ovm-toolchain')
```

Comment:
This should work no?
```suggestion
const * as ovm = from '@eth-optimism/ovm-toolchain'
```

Reply:
Not exactly sure. I tried to do that but it gave me some problems. Strange, but it looked like they messed up something with their exports. Will try again.

Reply:
Aha, they don't publish types, looks like only `.js` files are published.